CHANGES.md

```diff
@@ -1,3 +1,5 @@
 # xcube-gen-vito changes
 
 ## Changes
+
+* `VitoS2PlusInputProcessor` is now an XYInputProcessor
```

Comment:
This is not really acceptable, as your comment requires detailed knowledge about the current `XYInputProcessor` implementation. Please describe, which issue has been fixed or which functionlity has been added. CHANGES.md should be human-readable.